pkgs/development/tools/misc/texinfo/make-texinfo.nix

```
@@ -41,8 +47,8 @@ stdenv.mkDerivation {
 
   # ncurses is required to build `makedoc'
   # this feature is introduced by the ./cross-tools-flags.patch
-  NATIVE_TOOLS_CFLAGS = if crossBuildTools then "-I${getDev buildPackages.ncurses}/include" else null;
-  NATIVE_TOOLS_LDFLAGS = if crossBuildTools then "-L${getLib buildPackages.ncurses}/lib" else null;
+  NATIVE_TOOLS_CFLAGS = if crossBuildTools then "-I${lib.getDev buildPackages.ncurses}/include" else null;
+  NATIVE_TOOLS_LDFLAGS = if crossBuildTools then "-L${lib.getLib buildPackages.ncurses}/lib" else null;
```

```suggestion
  env = {
    NATIVE_TOOLS_CFLAGS = if crossBuildTools then "-I${lib.getDev buildPackages.ncurses}/include" else null;
    NATIVE_TOOLS_LDFLAGS = if crossBuildTools then "-L${lib.getLib buildPackages.ncurses}/lib" else null;
  };
```

P.S.: shouldn't they be `""` instead of `null`? Nulls are discouraged.

---

I took the liberty to use `lib.optionalString` instead of `if / else` any reasons against that, @AndersonTorres?